exampleCourse/questions/demo/autograder/python/square/server.py

```diff
@@ -1,6 +1,3 @@
-import numpy as np
-
-
 def generate(data):
     data["params"]["names_for_user"] = [
```

Comment:
Might be worth inlining some of these descriptions with the new element feature? Not strictly necessary but some of these files are getting cleaned out anyway

Reply:
I'll let that wait for another PR to keep this PR focused on linting changes.